### ISSUE TITLE
Home and End jump to the beginning and end of scene description as well as slider

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molarity#216]
**Test device**
Mac Book Pro
**Operating System**
10.15.2
**Browser**
Safari
**Problem description**
For https://github.com/phetsims/QA/issues/464. Found with MacOS VO.
When using home and end to jump to the ends of sliders, VO will read off the beginning or end of the scene description, instead of the new slider value. It then reads the new molarity. If you then move the slider (using the arrow keys), it will not read the new slider value but will read the new molarity. To fix this tab away and back to the slider.
**Steps to reproduce**
1. Turn on VO
2. Go to a slider
3. Use arrows to move slider up and down. Listen to normal reading
4. Press key button or end key on slider

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Molarity‬
URL: https://phet-dev.colorado.edu/html/molarity/1.5.0-rc.1/phet/molarity_all_phet.html
Version: 1.5.0-rc.1 2019-12-19 01:13:41 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_2) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.0.4 Safari/605.1.15
Language: en-US
Window: 1557x883
Pixel Ratio: 2/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0 (1.20)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 15 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 16)
Max viewport: 8192x8192
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by jessegreenberg** [comment]
Ah, I think we have seen this before and found it is something we cannot work around. Ill look.

**Comment by jessegreenberg** [comment]
I found it! https://github.com/phetsims/a11y-research/issues/149

It is an Apple bug and @zepumph submitted a bug report about it. Basically, Home and End keys will move the virtual cursor AND move the slider thumb.

I want to make sure that @terracoda is aware of this. Changing the PDOM HTML may resolve this. For example, removing an `aria-labelledby` in GFL resolved the issue. But I don't know if we want to mess with changing the HTML at this point. How large of an issue is this?

**Comment by terracoda** [comment]
This is about how VO navigates groups... we had this issue in GFL or GFLB.

**Comment by jessegreenberg** [comment]
I see, understood. Does this require further attention or can we note in screen reader bug tracker and close?

**Comment by terracoda** [comment]
What we did in GFL is remove the aria-labelledby which was basically crating the group.

**Comment by terracoda** [comment]
Apologies, I didn't read the issue title properly. Is this a problem?

**Comment by jessegreenberg** [comment]
In my opinion, it is a problem but not one we should spend time working around since it is mostly an issue with VoiceOver. I recommend that we make note of this in the screen reader bug tracker so we can ignore all issues like this in the future until Apple has it resolved. Is that OK with you?

**Comment by terracoda** [comment]
I'd just like to understand what @KatieWoe experienced a little better. 
Is there and aria-labelled by on the "Beaker Solution Controls" heading?

**Comment by jessegreenberg** [comment]
There are three aria-labelledby attributes in the Molarity PDOM. One on the ComboBox. One on the ComboBox list. And one on the "Sim Resources" navigation bar.

**Comment by terracoda** [comment]
I can't reproduce this issue. I'll try reading to the slider, and see if that does it.

**Comment by terracoda** [comment]
Nope, still can't reproduce. 
I don't know how @KatieWoe is getting to the slider. I tried go there with the tab key and with the cursor keys, and with the mouse.

I cannot reproduce. It might be related to version. 

@jessegreenberg, The `aria-labelledby`'s on the combobox are needed, and we don't ahve a problem with the navbar, so all is good.

Please do as you suggested, make a note, so QA is aware of this bug.

**Comment by jessegreenberg** [comment]
OK sounds good  - it may also depend on your keyboard @terracoda, I was able to reproduce on my mac by pressing "fn" + "right arrow" while focus was on the slider. But if you have a dedicated "end" key on your keyboard it may not happen.

**Comment by terracoda** [comment]
I was also using "fn" and arrow keys.

**Comment by terracoda** [comment]
Anyways, no worries. Please make the note and close :-)

**Comment by jessegreenberg** [comment]
Hmm, I am not sure then!  OK I made note in the screen reader bug tracker, closing this issue.
